src/modules/v1/character/dto/character.dto.ts

```diff
@@ -0,0 +1,78 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsBoolean,
+  IsDate,
+  IsNotEmpty,
+  IsNumber,
+  IsString,
+  MaxLength,
+  MinLength,
+} from 'class-validator';
+
+export class CharacterDTO {
+  @ApiProperty({
+    description: '캐릭터의 id 값',
+    required: true,
+  })
+  @IsNumber()
+  id: number;
+
+  @ApiProperty({
+    description: '캐츄의 이름입니다.',
+    required: true,
+  })
+  @IsNotEmpty()
+  @IsString({ message: '캐츄의 이름은 문자열이여야 합니다.' })
+  @MinLength(1, { message: '닉네임은 1자 이상 20자 이하이여야 합니다' })
+  @MaxLength(20, { message: '닉네임은 1자 이상 20자 이하이여야 합니다' })
```

Comment:
여기 닉네임 말고 캐츄 이름으로 해야할 듯 합니다!

```suggestion
  @MinLength(1, { message: '캐츄의 이름은 1자 이상 20자 이하이여야 합니다' })
  @MaxLength(20, { message: '캐츄의 이름은 1자 이상 20자 이하이여야 합니다' })
```